javascript/exorelay/src/websocket-connector/websocket-connector.ls

```diff
@@ -71,6 +71,13 @@ class WebSocketConnector extends EventEmitter
       ..on 'message', @_on-socket-message
       ..on 'error', @_on-socket-error
       ..on 'close', @_on-socket-close
+    @retry 100
```

Comment:
I don't think this prevents the swamp of error messages that gets logged as the library continually emits error messages which we log. We should only log those unable to connect errors with exponential backoff